test/integration/patients/patient/patient-flow.js

```diff
@@ -389,6 +404,37 @@ context('patient flow page', function() {
       .find('.comment__item')
       .should('have.length', 3);
 
+    cy.sendWs({
+      category: 'AttachmentAdded',
+      resource: {
+        type: 'patient-actions',
+        id: testFlowAction.id,
+      },
+      payload: {
+        clinician: {
+          type: 'clinicians',
+          id: '95014c50-2401-4d95-83e1-4a39fb928c90',
+        },
+        file: {
+          type: 'files',
+          id: '70294032-539b-4dc0-93df-9e4ff2eee8c1',
+        },
+        attributes: {
```

Comment:
so there's attributes would be the `resource` attributes... but these are the file attributes so they'd need to be nested in file or else we need to make a request.